### ISSUE TITLE
hls.js version

### ISSUE THREAD

**Issue by alienatorZ** [devcshort/react-hls#26]
The package.json shows this uses hls.js version ^0.14.17. hls.js is now at 1.0.0. 
Could the version be updated?

**Comment by devcshort** [comment]
> The package.json shows this uses hls.js version ^0.14.17. hls.js is now at 1.0.0.
> Could the version be updated?

Yup! I'll need a bit of time to work on it and look in to their migration guide. Thanks for bringing this up, didn't realize they'd be hitting the 1.0 release already.
